Docs: PPL support in Elasticsearch datasource (#29464)
grafana/grafana: docs/sources/datasources/elasticsearch.md

| `s`        | second      |
| `ms`       | millisecond |

> ### Piped processing language (PPL) support

Reviewer: ```suggestion
### Piped Processing Language (PPL) support
```

Reviewer: Capitalize proper nouns

> PPL is available starting opendistro_sql 1.11.0.0. PPL support for Grafana can be enabled in the datasource configuration setting when Elasticsearch version is set to 7.0+. See [Open Distro for Elasticsearch SQL plugin](https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/#sql) for plugin installation guide and [plugin compatibility](https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/#plugin-compatibility) for version compatibility information.

Reviewer: ```suggestion
PPL is available starting opendistro_sql 1.11.0.0. PPL support for Grafana can be enabled in the data source configuration setting when Elasticsearch version is set to 7.0+. See [Open Distro for Elasticsearch SQL plugin](https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/#sql) for plugin installation guide and [plugin compatibility](https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/#plugin-compatibility) for version compatibility information.
```

Reviewer: Per the [Grafana documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md), "data source" is always two words.

> If PPL support is enabled, the Elasticsearch query editor will give you the option of using PPL query syntax instead of Lucene.

Reviewer: ```suggestion
If PPL support is enabled, then the Elasticsearch query editor provides the option of using PPL query syntax instead of Lucene.
```

> The Elasticsearch query editor allows you to select multiple metrics and group by multiple terms or filters. Use the plus and minus icons to the right to add/remove

Reviewer: ```suggestion
The Elasticsearch query editor allows you to select multiple metrics and group by multiple terms or filters. Use the plus and minus icons to the right to add or remove
```

metrics or group by clauses. Some metrics and group by clauses haves options, click the option text to expand the row to view and edit metric or group by options.

> ## PPL Query editor

Reviewer: ```suggestion
## PPL query editor
```

Reviewer 2: Thank you for the review! All the edits should be addressed now. 
Quick question, I capitalized the PPL Query editor as it was done so in the above section "Metric Query editor". Please let me know if you would like that one to be also uncapitalized.

Reviewer: If it is in the UI, then match the UI.

Otherwise, all headings should be title case, so uncapitalize both. (Sorry, I am not super familiar with Elastic or the UI, so if I am wrong and it is in the UI, let me know.)

Reviewer 2: That makes sense, thank you! PPL query editor is in the UI.

> The Elasticsearch query editor allows you to write PPL queries if PPL support is enabled. Select PPL in the dropdown menu next to the query input field to switch to the PPL query editor.

Reviewer: ```suggestion
The Elasticsearch query editor allows you to write PPL queries. Select PPL in the drop-down menu next to the query input field to switch to the PPL query editor.
```

> PPL queries can be written in the query input field.

Reviewer: ```suggestion

Write PPL queries in the query input field.

```

Reviewer: Try to avoid sentences in passive voice. Active voice is clearer and uses fewer words.

> If the input field is empty, a default query of `source=<index>` will be used, where `<index>` is the index configured in the data source settings.

Reviewer: ```suggestion
If the input field is empty, then a default query of `source=<index>` is used, where `<index>` is the index configured in the data source settings.
```

Reviewer: Try to write in present tense rather than future.

> There are three query format options available: `Table`, `Logs`, and `Time series`.

Reviewer: ```suggestion
There are three query format options available: 
- **Table -** Accepts any PPL query.
- **Logs -** Accepts any PPL query.
- **Time series -** The PPL query must return two fields. One field must have a `date`, `time`, `datetime`, or `timestamp` data type. The other field must contain a value with a numeric data type.
```

> For `Table` and `Logs` formats, any PPL query can be used. For `Time series` format, the PPL query must return two fields.

Reviewer: ```suggestion
```

> One field must have a type of either `date`, `time`, `datetime`, or `timestamp`. The other field must have numeric datatype as values.

Reviewer: ```suggestion
```
